src/Bridge/Doctrine/Orm/State/ItemProvider.php

```diff
@@ -75,6 +81,10 @@ public function provide(string $resourceClass, array $identifiers = [], ?string
             }
         }
 
+        if (\count($queryBuilder->getQuery()->getResult()) > 1) {
+            throw new \Exception();
```

Comment:
It would be nice to have an explicit message here.

Reply:
I forgot to remove it, I used it to debug but the Exception is thrown by `getOneOrNullResult `